validation-test/StdlibUnittest/ChildProcessShutdown/FailIfChildCrashesDuringShutdown.swift

```diff
@@ -2,7 +2,7 @@
 // REQUIRES: executable_test
 
 import StdlibUnittest
-#if os(Linux) || os(FreeBSD) || os(PS4) || os(Android)
+#if os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Windows)
```

Comment:
Not tested on windows-msvc. Only tested on Cygwin, and I believe it will be pass for MinGW.

Did you build an alternatives to Glibc for MSVC ?